Source/SwiftLintFramework/Models/Configuration.swift

```diff
@@ -17,10 +17,26 @@ public struct Configuration: Hashable {
         case allEnabled
     }
 
+    public enum IndentationMode: Equatable {
+        case tabs
+        case spaces(count: Int)
+
+        // MARK: Equatable
+
+        public static func == (lhs: IndentationMode, rhs: IndentationMode) -> Bool {
+            switch (lhs, rhs) {
+            case (.tabs, .tabs): return true
+            case (.spaces(let lhs), .spaces(let rhs)): return lhs == rhs
+            case (_, _): return false
+            }
+        }
+    }
+
     // MARK: Properties
 
     public static let fileName = ".swiftlint.yml"
 
+    public let indentationMode: IndentationMode        // mode to use when indenting
```

Comment:
I chose the name `IndentationMode ` because it was similar to `RulesMode`, is that alright? Or does the `Mode` suffix imply something that isn't valid for this enum?

Reply:
IMO The mode suffix works for the rule mode, but not for the indentation setting.